com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -69,9 +71,27 @@ public class Bob {
     public static final String VARIANT_RELEASE = "release";
     public static final String VARIANT_HEADLESS = "headless";
 
-    private static boolean verbose = false;
```

Comment:
This flag was not really used anymore in 1.4.4 but we forgot to remove it when we switched to java.util.logging in bob